themes/wilson_theme_starterkit/wilson_theme_starterkit.theme

```diff
@@ -350,7 +353,7 @@ function _wilson_theme_starterkit_background_styles($imageUri, $elementId, $styl
   // Generate the initial fallback background styles.
   $fallbackStyleName = $styleNamePrefix . 'fallback';
   if (ImageStyle::load($fallbackStyleName)) {
-    $imageUrl = Markup::create(file_url_transform_relative(ImageStyle::load($fallbackStyleName)->buildUrl($imageUri)));
```

Comment:
I'm surprised that this hasn't been flagged in deprecation reports, but the `ImageStyle::load` methods should generally be replaced with entity type manager lookups: 

```php
$entity_type_manager = Drupal::entityTypeManager();
$image_style_storage = $entity_type_manager->loadStorage('image_style');
$style = $image_style_storage->load($fallbackStyleName);
```

... would remove the need for the `use Drupal\image\Entity\ImageStyle;` line at the top of the file too.

This is true anywhere you're loading, creating or otherwise interacting with entities using the entity classes.

This change doesn't need to be made as part of this PR, but it would be good to handle it before a stable Drupal 10 version of Wilson is released!

Reply:
Good spot, Cheers @iamsophiesk.

Reply:
@AccessPaul, we can update this now or later. 👍🏻

Reply:
Thanks for picking up on this @iamsophiesk. I've implemented this but with using `$entity_type_manager->getStorage('image_style')` as `loadStorage()` doesn't seem to exist.

Reply:
> Thanks for picking up on this @iamsophiesk. I've implemented this but with using `$entity_type_manager->getStorage('image_style')` as `loadStorage()` doesn't seem to exist.

I always try one and then remember that I meant the other :joy: thanks for implementing this so soon.